app/services/publication-service.js

```diff
@@ -321,4 +327,38 @@ export default class PublicationService extends Service {
       CONSTANTS.PUBLICATION_STATUSES.PUBLICATION_REQUESTED
     );
   }
+
+  async setRegulationTypeThroughReferenceDocument(
```

Comment:
Aangezien deze method enkel gebruikt wordt in de `Publications::PublicationBatchEditModal` component, zou ik hem daar definiëren ipv op de publication-service. Enkel `referenceDocument` als argument zou ook voldoende moeten zijn. Publication flow kan dan bekomen worden uit `referenceDocument.publicationFlow`, wat een promise is die onmiddellijk zal resolven op dat moment.